src/Components/PokemonCard.js

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import axios from "axios";
+import {useEffect} from "react";
+
+function PokemonCard({name,classname}) {
+
+    const [pokemonData, setPokemonData] = useState({})
+
+    useEffect(()=>{
+        async function fetchData(){
+            try{
+                const result = await axios.get(`https://pokeapi.co/api/v2/pokemon/${name}`)
+                console.log(result)
+                setPokemonData(result)
+            } catch (e){
+                console.error(e)
+            }
+        }
+        fetchData()
+    },[])
+    return (
+        <>
+            {Object.keys(pokemonData).length > 0 &&
```

Comment:
Dan kan deze check wat simpeler
```suggestion
            {pokemonData &&
```